src/core/reopt_inputs.jl

```diff
@@ -35,6 +35,8 @@ struct REoptInputs <: AbstractInputs
     ratchets::UnitRange
     techs_by_exportbin::Dict{Symbol, AbstractArray}  # keys can include [:NEM, :WHL, :CUR]
```

Comment:
This comment says :CUR but I think it say :EXC, right?
Did you intentionally not include :EXC for the battery and if so why?